routers/repo/wiki.go

```diff
@@ -350,7 +350,7 @@ func NewWikiPost(ctx *context.Context, form auth.NewWikiForm) {
 		return
 	}
 
-	ctx.Redirect(ctx.Repo.RepoLink + "/wiki/" + models.WikiNameToFilename(wikiName))
+	ctx.Redirect(ctx.Repo.RepoLink + "/wiki/" + wikiName)
```

Comment:
I think `WikiNameToSubURL` should be here.

Reply:
Thanks! I missed this function completely. :-) I have pushed an updated version of the PR.